sys-libs/ncurses/ncurses6-6.1.recipe

```diff
@@ -0,0 +1,224 @@
+SUMMARY="New curses library"
+HOMEPAGE="https://www.gnu.org/software/ncurses/ncurses.html"
+SOURCE_URI="https://ftpmirror.gnu.org/ncurses/ncurses-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/ncurses/ncurses-$portVersion.tar.gz"
+CHECKSUM_SHA256="aa057eeeb4a14d470101eff4597d5833dcef5965331be3528c08d99cebaa0d17"
+SOURCE_DIR="ncurses-$portVersion"
+LICENSE="MIT"
+COPYRIGHT="1998-2017 Free Software Foundation, Inc."
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64 arm"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PATCHES="ncurses-6.1.patchset"
```

Comment:
Shall we sort some variables and introduce a few custom variables?
```diff
 HOMEPAGE="https://www.gnu.org/software/ncurses/ncurses.html"
+COPYRIGHT="1998-2017 Free Software Foundation, Inc."
+LICENSE="MIT"
+REVISION="1"
 SOURCE_URI="https://ftpmirror.gnu.org/ncurses/ncurses-$portVersion.tar.gz
 	https://ftp.gnu.org/gnu/ncurses/ncurses-$portVersion.tar.gz"
 CHECKSUM_SHA256="aa057eeeb4a14d470101eff4597d5833dcef5965331be3528c08d99cebaa0d17"
 SOURCE_DIR="ncurses-$portVersion"
-LICENSE="MIT"
-COPYRIGHT="1998-2017 Free Software Foundation, Inc."
-REVISION="1"
+PATCHES="ncurses-$portVersion.patchset"
+
 ARCHITECTURES="x86_gcc2 x86 x86_64 arm"
 SECONDARY_ARCHITECTURES="x86_gcc2 x86"
  
-PATCHES="ncurses-6.1.patchset"
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%.*}"
 
 PROVIDES="
-	ncurses6$secondaryArchSuffix = $portVersion compat >= 6.1
+	ncurses6$secondaryArchSuffix = $portVersionCompat
-	lib:libform$secondaryArchSuffix = $portVersion compat >= 6.1
+	lib:libform$secondaryArchSuffix = $libVersionCompat
```
(same changes for the other `lib:` lines.)

Note: I'm not sure about `libVersionCompat`. Maybe we should use 
```
libVersionCompat="$libVersion compat >= ${libVersion%.*}"
```
instead of
```
libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```